plugins/recorded_future/help.md

```diff
@@ -2555,7 +2580,7 @@ Example output:
 
 # Version History
 
-* 6.0.2 - Fixed schemas | Updated the SDK to latest version | Refreshed with latest plugin tooling
+* 7.0.0 - Fixed schemas | Updated the SDK to latest version | Refreshed with latest plugin tooling | Added Memory Error handling for Risk Lists | Added `Risk List GZip` output to Risk List actions
```

Comment:
Is it  worth adding something in the troubleshooting section to state the behaviour around the risk lists i.e. returning a gzip file if its too large?